app/controllers/v1/songs_controller.rb

```diff
@@ -1,7 +1,17 @@
 class V1::SongsController < ApplicationController
+
+
   def index
+    Song.create(name: 'Mia', artist: 'Jas', spotify_uri: 'test')
```

Comment:
so we won't want to create test data right in our `index` action like this.  If we did, every time someone hits our `GET /songs` endpoint, we'll add a new song to the DB, when really all we want to do is fetch songs.

If you need records for testing, use FactoryGirl to create the records directly in your tests :)